### ISSUE TITLE
Grid size

### ISSUE THREAD

**Issue by zalzalzal17** [jobobby04/TachiyomiSY#1303]
### Steps to reproduce

Change grid size row on display tab

### Expected behavior

Grid size can change from 1 per row to 10 per row

### Actual behavior

The grid size 4 per row cant work, when changing from 3 per row it jump to 5 per row at potrait mode, landscape mode work as normal

### TachiyomiSY version

1.11.0

### Android version

Android 14

### Device

Mi 11T Pro

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https:/mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[1.11.0](https://github.com/jobobby04/tachiyomisy/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by cuong-tran** [comment]
My SY (preview) can show 4 per row perfectly in portrait mode, both on small/big screen (a Fold)

**Comment by zalzalzal17** [comment]
> My SY (preview) can show 4 per row perfectly in portrait mode, both on small/big screen (a Fold)

Yeah its work normal before i updated the app, just some small bug nothing more

**Comment by voltrare** [comment]
Same here it defaults 2 per row and doesnt change even if I adjust the slider

**Comment by Smol-Ame** [comment]
This has been discussed before in the Mihon server.

It's a known issue (on Mihon too) with screen scaling/aspect ratio that affects the slider's padding or whatever. Current possible workarounds:
* Tap on the location where the desired grid size might be, you might be able to select it by tapping rather than sliding
* Change your screen zoom size (like Phone Settings > Display > Screen zoom) or resize the app's screen (if possible)
